### PR TITLE
中英文文档合并以及Hyperledger文件重命名

### PR DESCRIPTION
**Pull request by davidkhala**
格式更新
大型改动

### CODE REVIEW

**Comment by SamYuan1990** [comment]
@davidkhala , I hope in this pr we can have some document upgrade as mentioned in https://github.com/Hyperledger-TWGC/tape/issues/116

**Comment by davidkhala** [comment]
@SamYuan1990 but #116 has ambitious beyond tape as probe is also included.

**Comment by SamYuan1990** [comment]
> @SamYuan1990 but #116 has ambitious beyond tape as probe is also included.

I see, or could you please help with https://github.com/Hyperledger-TWGC/tape/issues/131 ? To be honest... I hope we can have a pr with large scope upgrade for document.

**Comment by guoger** [comment]
@davidkhala commit需要sign-off

File: README.md [hunk 1]
```diff
@@ -120,28 +58,18 @@ Here are possbile values (warn by default)
...
-![tape workflow](./docs/images/tape.jpeg)
-</details>
 
-## Maintainers
 
-| Name   | mail                     | github-ID               
-| ------ | ------------------------ | ----------- | 
-| Jay Gou | guojiannan1101@gmail.com | guoger      |
-| Sam Yuan   | yy19902439@126.com       | SamYuan1990 | 
-| Stone Cheng   | chengyang418@163.com     | stone-ch    |
```
Comment (guoger) [inline README.md:147]: not sure why we need to delete this section? IMO this makes readme more complete

**Review by guoger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (davidkhala) [reply README.md:147]: I planned to move this section to MAINTAINERS.md. 
How about we set the title ##Maintainers here as a link to MAINTAINERS.md ?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Maintainers 维护者信息
+
```
Comment (davidkhala) [inline MAINTAINERS.md:2]: @guoger 郭哥，维护者移到这里了

**Comment by davidkhala** [comment]
@SamYuan1990 看起来fabric-samples 的脚本又改了
https://dev.azure.com/Hyperledger/TWGC/_build/results?buildId=30217&view=logs&j=8d03cc15-4df7-568b-60a2-97dabd397d21&t=63bb61fd-6838-5eea-5642-3155fa8636f7&l=548

**Review by stone-ch — DISMISSED** [review]
LGTM
Make the README clean. And actually it is necessary to add the complete English documents, too.

**Review by stone-ch — DISMISSED** [review]

**Comment by davidkhala** [comment]
@stone-ch 刚才git reset的时候选错了方式，把代码搞乱了，现在应该好了

**Review by stone-ch — APPROVED** [review]

**Review by guoger — APPROVED** [review]

**Comment by davidkhala** [comment]
close #116

**Pull request merged**